src/libsync/account.h

```diff
@@ -168,23 +168,18 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject
 
     bool hasCapabilities() const;
 
+    // product
+    QString serverProduct() const;
```

Comment:
How about `serverProductName`?

The comment is redundant and doesn't add information, so it should be removed again.